Charging test failure

```
_______________________________________________________________________________________________________________________________________________________________ TestDesatAction.test_desat_action_pointing ________________________________________________________________________________________________________________________________________________________________

self = <test_int_actions.TestDesatAction object at 0x2bbdaa230>

    def test_desat_action_pointing(self):
        env = self.make_env()
        env.unwrapped.satellite.sat_args_generator["desatAttitude"] = "nadir"
        env.reset(seed=0)
        env.step(0)  # Desat
        battery_level_nadir = env.unwrapped.satellite.dynamics.battery_charge
    
        env.unwrapped.satellite.sat_args_generator["desatAttitude"] = "sun"
        env.reset(seed=0)
        env.step(0)  # Desat
        battery_level_sun = env.unwrapped.satellite.dynamics.battery_charge
>       assert battery_level_sun > battery_level_nadir
E       assert 195414.9288685548 > 212166.30922296405

tests/integration/act/test_int_actions.py:167: AssertionError
```

--- a/tests/integration/act/test_int_actions.py
+++ b/tests/integration/act/test_int_actions.py
@@ -121,6 +121,7 @@ class TestDesatAction:
                 sat_args=self.DesatSat.default_sat_args(
                     oe=random_orbit,
                     wheelSpeeds=[1000.0, -1000.0, 1000.0],
+                    nHat_B=np.array([0, 1, 0]),
                 ),
             ),
             scenario=UniformTargets(n_targets=0),

